R/get_abcd_raw.R

```diff
@@ -3,26 +3,38 @@
 #' A longer description of the function
 #'
 #' @param portfolio_type A description of the argument
+#' @param data_path Path to directory containing ABCD Data files
+#' @param start_year Start year of the analysis
+#' @param time_horizon Length of the analysis in years (analysis will run from
+#' `start_year` to `start_year + time_horizon`)
+#' @param sector_list (character) A vector of PACTA sectors to include in the
+#' analysis.
 #'
 #' @return A description of the return value
 #'
 #' @export
 
-get_abcd_raw <- function(portfolio_type) {
+get_abcd_raw <- function(
+  portfolio_type,
+  data_path,
```

Comment:
```suggestion
  analysis_inputs_path,
```

Reply:
`analysis_inputs_path` is used all over the place across multiple repos... I don't want to introduce an additional name for it, bound to cause confusion